src/Uno.UI.RemoteControl.Server.Processors/HotReload/MetadataUpdates/CompilationWorkspaceProvider.cs

```diff
@@ -81,6 +93,23 @@ public static void InitializeRoslyn()
 			}
 		}
 
+		private static Version GetDotnetVersion()
+		{
+			var result = ProcessHelper.RunProcess("dotnet.exe", "--version");
```

Comment:
Is this Windows-specific code path?

Reply:
No, the underlying runtime is able to transform this a Unix binary name automatically (see the method below).